core/cog-platform.c

```diff
@@ -125,9 +125,19 @@ cog_platform_init_web_view (CogPlatform   *platform,
     CogPlatformClass *klass = COG_PLATFORM_GET_CLASS(platform);
     if (klass->init_web_view) {
         if (klass->get_view_type) {
+#if GLIB_CHECK_VERSION(2, 63, 1)
```

Comment:
GLib 2.63.1 was released in October 2019. That's more than four years ago... so now I am wondering in which environment this is a problem.

I suggest we add a check in the top-level `meson.build` file to make sure the GLib version is new enough.

Reply:
On Yocto Dunfell with long time support until April of 2024 [[ref](https://wiki.yoctoproject.org/wiki/Releases)] so you could expect this issue affecting to environments still supported in [meta-webkit](https://github.com/Igalia/meta-webkit/blob/main/conf/layer.conf#L20). If you think this is not a good argument, I could move it as an adaptation patch inside meta-webkit.

Reply:
Ahm, fair. Let's merge this and bump the GLib dependency in the next cycle after March-April 2024.